client/js/jsonedit.js

```diff
@@ -38,7 +38,7 @@ const jeCommands = [
     id: 'je_colorPicker',
     name: 'change color',
     options: [ { type: 'color', label: 'color' } ],
-    context: '.*#([0-9a-fA-F]{8}|[0-9a-fA-F]{6}|[0-9a-fA-F]{4}|[0-9a-fA-F]{3})|^.* ↦ color$' + String.fromCharCode(36),
+    context: '.*#([0-9a-fA-F]{8}|[0-9a-fA-F]{6}|[0-9a-fA-F]{4}|[0-9a-fA-F]{3})|^.* ↦ color',
```

Comment:
Correct should be the middle ground: `'.*#([0-9a-fA-F]{8}|[0-9a-fA-F]{6}|[0-9a-fA-F]{4}|[0-9a-fA-F]{3})|^.* ↦ color' + String.fromCharCode(36)`

Doesn't really matter though if the context ends with `color` or if something is after that so :shrug:.

Reply:
> > No example available for populating input (color always start at #ff0000)
> 
> If you want to preset the input with the current value of the property you're editing, then I don't think that's possible yet.

then this PR is ready. it's already better then what we currently have